### ISSUE TITLE
Weathered and cut copper blocks do not have #forge:storage_blocks/copper tag

### ISSUE THREAD

**Issue by SaltymcSaltface** [EnigmaticaModpacks/Enigmatica9#649]
### Modpack Version

1.10.2

### Describe your issue.

Weathered and cut copper blocks do not have #forge:storage_blocks/copper tag.
This is just an annoying issue, not high priority.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by DAmNRelentless** [comment]
It's intended. What exactly is the "annoying issue" this causes? How does this affect your gameplay?
Giving these blocks the storage block tag would cause a dupe bug in the stone cutter and cause unification issues as described here: https://github.com/EnigmaticaModpacks/Enigmatica8/issues/283#issuecomment-1257269746

**Comment by MuteTiefling** [comment]
Considering the tag isn't meant to be there, I'll go ahead and close this out. However, do let us know what the actual issue is you're trying to solve. There may be another way of improving it.

**Comment by SaltymcSaltface** [comment]
The annoying issue is I have hundreds of oxidized/weathered copper blocks that I harvested from an ocean monument, and they cannot be broken down into copper ingots without running them through an enrichment chamber. I ended up just doing that, but it doesn't make sense to not be able to just convert to ingots, as a cubic meter of copper would not oxidize more than .2mm before becoming inert.
The cut blocks duplication issue sounds like a problem that should be solved within the stonecutter recipe table, as being able to easily convert between blocks and ingots is a fundamental feature in Minecraft.
